core/arch/x86/decode_table.c

```diff
@@ -3176,10 +3177,14 @@ const instr_info_t prefix_extensions[][12] = {
     {OP_mulss, 0xf30f5910, "mulss", Vss, xx, Wss, Vss, xx, mrm, x, END_LIST},
     {OP_mulpd, 0x660f5910, "mulpd", Vpd, xx, Wpd, Vpd, xx, mrm, x, END_LIST},
     {OP_mulsd, 0xf20f5910, "mulsd", Vsd, xx, Wsd, Vsd, xx, mrm, x, END_LIST},
-    {OP_vmulps, 0x0f5910, "vmulps", Vvs, xx, Hvs, Wvs, xx, mrm|vex, x, END_LIST},
-    {OP_vmulss, 0xf30f5910, "vmulss", Vdq, xx, Hdq, Wss, xx, mrm|vex, x, END_LIST},
-    {OP_vmulpd, 0x660f5910, "vmulpd", Vvd, xx, Hvd, Wvd, xx, mrm|vex, x, END_LIST},
-    {OP_vmulsd, 0xf20f5910, "vmulsd", Vdq, xx, Hdq, Wsd, xx, mrm|vex, x, END_LIST},
+    {OP_vmulps, 0x0f5910, "vmulps", Vvs, xx, Hvs, Wvs, xx, mrm|vex, x, tpe[25][8]},
```

Comment:
Though maybe it's best to not duplicate what you're doing, so that spot-checks look at it differently.

Reply:
Sounds like you're checking whether I am checking: For non-round-trip checks via create macros, I am assembling basic versions of the opcodes that are added, compile them, and then run them through DR and check in DR's log for correctness (DR syntax). I am doing this only with basic forms of the instruction, xmm and zmm registers: one time with xmm0, xmm1, etc. and another time with the AVX-512 extended registers > xmm16. No other checks have been done in addition to round-trips.

Reply:
But in #3629 you said you were comparing to objdump.

Reply:
I am not comparing Intel syntax. It is not compared by script. I am not testing mask k1 - k7 currently because the operand type (TYPE_K_EVEX) has been working. k0 is not printed.

Reply:
I am looking for a way to do a good review: with the evex encoding the bytes field in the tables is no longer human-readable and the only feasible way seems to be to take the compiled code output.

Reply:
Fair enough. Maybe continue offline.